packages/jest-config/src/__tests__/__snapshots__/resolveConfigPath.test.ts.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[` 1`] = `
+"● Multiple configurations found:
+
+Jest found configuration in \`/var/folders/14/qp47l23n64j5pyd7vllb78c00000gn/T/resolve_config_path_test/a/b/c/jest.config.js\`, \`/var/folders/14/qp47l23n64j5pyd7vllb78c00000gn/T/resolve_config_path_test/a/b/c/jest.config.ts\`, \`/var/folders/14/qp47l23n64j5pyd7vllb78c00000gn/T/resolve_config_path_test/a/b/c/jest.config.mjs\`, \`/var/folders/14/qp47l23n64j5pyd7vllb78c00000gn/T/resolve_config_path_test/a/b/c/jest.config.cjs\` and \`/var/folders/14/qp47l23n64j5pyd7vllb78c00000gn/T/resolve_config_path_test/a/b/c/jest.config.json\`.
```

Comment:
this path is probably unique per run - do a `replace` for `DIR` in the string before snapshotting it. just a `.replace(new RegExp(DIR, 'g'), '<rootDir>') or something

Reply:
Shoot, thanks.

And do you know a nice way to do it? I came up with `try catch replace throw`, is there a better way?

Reply:
oh, this is from the unit tests - I meant to snapshot in the integration tests 🙂 

The unit tests here can probably be `toThrowError('Multiple configurations found')` - we don't care about the exact look of the message in this case

Reply:
Ok. I thought it is more appropriate to add them to unit tests, but I will move them.

Reply:
keep both, but we generally assert on how things look from the user's perspective in E2E tests 🙂

Reply:
I moved the snapshots to e2e tests. (Normal unit tests are still there).